schemas/scanorganic-README.md

```diff
@@ -0,0 +1,99 @@
+﻿# EDDN ScanOrganic Schema
+
+## Introduction
+Here we document how to take data from an ED `ScanOrganic` Journal Event and
+properly structure it for sending to EDDN.
+
+Please consult [EDDN Schemas README](./README-EDDN-schemas.md) for general
+documentation for a schema such as this.
+
+If you find any discrepancies between what this document says and what is
+defined in the relevant Schema file, then you should, in the first instance,
+assume that it is the Schema file that is correct.
+**PLEASE open
+[an issue on GitHub](https://github.com/EDCD/EDDN/issues/new/choose)
+to report any such anomalies you find so that we can check and resolve the
+discrepancy.**
+
+## Senders
+The primary data source for this schema is the ED Journal event `ScanOrganic`.
+
+### ScanType
+The 'Analyse' scan type only triggers when you have the scan tool out for a
+long enough duration following your third scan. It's possible to put away the
+tool before this completes and the event will then trigger the next time the
+tool is used, which could be in another system entirely. For this reason,
+it may report incorrect data and is thus excluded from submission.
+
+### Variant
+Variant should be reported if present. It was not included in older journal
+versions and therefore is not required to facilitate older journal submissions.
+
+### Use of status.json
+You are encouraged to augment your submission with values from status.json. When status.json is used:
+
+* You must verify that the timestamp from status.json is no more than one minute older than the timestamp from the `ScanOrganic` journal event
+* You must verify that the status.json `Flags2` value `0x00000010` is true (indicating that the commander is on-foot on a planet)
+* You must verify that the `BodyName` from status.json and journal events are in agreement
+* Please augment your submission with `BodyName`, `Latitude`, and `Longitude` values
+
+### Augmentations
+#### horizons and odyssey flags
+Please read [horizons and odyssey flags](../docs/Developers.md#horizons-and-odyssey-flags)
+in the Developers' documentation.
+
+#### gameversion and gamebuild
+You **MUST** always set these as per [the relevant section](../docs/Developers.md#gameversions-and-gamebuild)
+of the Developers' documentation.
+
+#### StarSystem
+You MUST add a StarSystem key/value pair representing the name of the system
+this event occurred in. Source this from either Location, FSDJump or
+CarrierJump as appropriate.
+
+**You MUST apply a location cross-check, as per
+[Other data augmentations](../docs/Developers.md#other-data-augmentations).**
+
+#### StarPos
+You MUST add a `StarPos` array containing the system co-ordinates from the
+last `FSDJump`, `CarrierJump`, or `Location` event.
+
+**You MUST apply a location cross-check, as per
+[Other data augmentations](../docs/Developers.md#other-data-augmentations).**
```

Comment:
These highlighted lines describe augments which are not currently allowed per the schema?